src/snowflake/snowpark/dataframe.py

```diff
@@ -451,14 +452,37 @@ def collect(
             statement_params=statement_params
         )
 
+    @df_collect_api_telemetry
+    def collect_nowait(
+        self,
+        *,
+        statement_params: Optional[Dict[str, str]] = None,
+        data_type: str = "row",
```

Comment:
this argument is not provided by the user: collect() will return rows and to_pandas() will return a pandas dataframe.

Reply:
yes, i am think about passing this arg to asyncjob instance inside the corresponding function like collect() that calls it. In this way, users will not see or operate on this arg